Makefile

```
@@ -4,7 +4,7 @@
 BUF_VERSION = 1.15.0
 GO_VERSION := $(shell awk '/^go / {print $$2}' go.mod)
 PROTOC_GEN_GO_VERSION := $(shell awk '/google.golang.org\/protobuf/ {print substr($$NF, 2)}' go.mod)
-
+PROTOC_ES_GEN_VERSION := $(shell jq -r '.dependencies["@bufbuild/protoc-gen-es"]' es/package.json)
```

nit. perhaps you mean
```suggestion
PROTOC_GEN_ES_VERSION := $(shell jq -r '.dependencies["@bufbuild/protoc-gen-es"]' es/package.json)
```

---

Thank you for your comments! I've addressed all of your comments by [7fc0970](https://github.com/cybozu/protobuf/pull/23/commits/7fc0970563c351fa64f82cba98c7561a71b1c573) and [8a9517f](https://github.com/cybozu/protobuf/pull/23/commits/8a9517f86ba0f490ed80e90189d405ce4539fb6d)